package.json

```diff
@@ -74,7 +74,7 @@
         "vite-plugin-dts": "^2.1.0"
     },
     "peerDependencies": {
-        "ol": "^7.1.0"
+        "ol": "^8.1.0"
```

Comment:
I guess this would be better

`"ol": "> 7.x <= 8.x"`

Reply:
```suggestion
        "ol": "> 7.x <= 8.x"
```

Reply:
Right! I updated the commit in this PR.